### PR TITLE
feat: traefic hpa

### LINKED ISSUE #705: Traefik HPA
### Overview

There is currently HPA set up, but it is scaling on cpu usage alone. I was just noticing our traefik pods are really memory throttled and should also scale up based on mem usage if needed as well.

### References and additional details

image

### Acceptance Criteria

- Trigger HPA for traefik on ~50% mem usage

### Need By Date

_No response_

### Screenshots/Mockups

_No response_

### Estimated effort

_No response_

### PR DESCRIPTION
**Pull request by lmyslinski**
Closes #705

### CODE REVIEW

File: pulumi/package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "clean": "rm -rf dist node_modules"
   },
   "dependencies": {
-    "@shapeshiftoss/cluster-launcher": "^0.18.0",
+    "@shapeshiftoss/cluster-launcher": "^0.19.0",
+    "axios": "^0.27.2",
```
Comment (kaladinlight) [inline pulumi/package.json:13]: I hoisted this dependency up so we don't need to include it here anymore
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -828,8 +828,9 @@ jobs:
             pulumi config set --path unchained:common.eks.nodeGroups[0].instanceTypes[3] r6i.4xlarge
             pulumi config set --path unchained:common.eks.nodeGroups[0].instanceTypes[4] r6id.4xlarge
             pulumi config set --path unchained:common.eks.traefik.autoscaling.enabled true
-            pulumi config set --path unchained:common.eks.traefik.autoscaling.cpuThreshold 20
+            pulumi config set --path unchained:common.eks.traefik.autoscaling.memoryThreshold 50
```
Comment (kaladinlight) [inline .circleci/config.yml:831]: want to just add `cpuThreshold` as well so we can explicitly see the config and not have to know/remember defaults?

File: .circleci/config.yml [hunk 3]
```diff
@@ -828,8 +828,9 @@ jobs:
...
             pulumi config set --path unchained:common.eks.traefik.autoscaling.maxReplicas 20
+            pulumi config set --path unchained:common.eks.traefik.autoscaling.minReplicas 3
```
Comment (kaladinlight) [inline .circleci/config.yml:833]: is replica count ignored when using the auto scaler? feel like we should just use `5` to match the prev replica count

Comment (lmyslinski) [reply .circleci/config.yml:833]: it affects just the initial deployment

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**